### DISCUSSION TITLE
Where are all old discussions?

### DISCUSSION THREAD

**Discussion by pauvt** [WinMerge/winmerge#511] [Q&A]
All links to http://forums.winmerge.org/ now are redirected to github/discussions.

Where are all old discussions?

**Comment by gerundt** [comment]
Were there many important discussions? Most looked outdated.

**Comment by Pierre-de-la-Verre** [comment]
Usually, old forums are kept, locked but available for users. If outdated or not should on the decision of the users.

**Reply by draney98** [reply]
Would also like to see some of the old answers back, if possible.

**Comment by ruffin--** [comment]
Ping. Any archive of the old forum available? If the answer is they're all dead, that's good info to have too!
